docs/library/micropython.rst

```diff
@@ -112,5 +112,14 @@ Functions
    the heap may be locked) and scheduling a function to call later will lift
    those restrictions.
 
+   Note: If ``schedule`` is called from a preempting IRQ, when memory
```

Comment:
Can you please use the xref syntax as described in https://github.com/micropython/micropython/blob/master/CODECONVENTIONS.md#documentation-conventions (specifically, all occurrences should be:
~~~
`schedule()`
~~~